cloud-nio/cloud-nio-impl-drs/src/main/scala/cloud/nio/impl/drs/DrsPathResolver.scala

```diff
@@ -78,13 +78,13 @@ case class Url(url: String)
 
 case class ChecksumObject(checksum: String, `type`: String)
 
-case class DosDataObject(size: Option[Long],
+case class DrsDataObject(size: Option[Long],
                          checksums: Option[Array[ChecksumObject]],
                          updated: Option[String],
                          urls: Array[Url])
 
-case class DosObject(data_object: DosDataObject)
+case class DrsObject(data_object: DrsDataObject)
 
 case class SADataObject(data: Json)
 
-case class MarthaResponse(dos: DosObject, googleServiceAccount: Option[SADataObject])
+case class MarthaResponse(drs: DrsObject, googleServiceAccount: Option[SADataObject])
```

Comment:
It looks like Martha is still returning key as `dos` instead of `drs` in it's response. This and it's references should be reverted back to `dos`.

Reply:
If I'm understanding this correctly, Martha is returning a field called `dos` with a DRS URL in it? If that's right I think my preference would be `case class MarthaResponse(dos: DrsObject, ...)`